### PR TITLE
docs(internal-plugin-mercury): require url in proxy example

### PR DESCRIPTION
**Pull request by jpjpjp**
# Pull Request Template

## Description

Slight modification to the proxy sample in the readme.  I stumbled over the fact that the sample did not require the url package.   WIth that said my modifications now get secrets out of the environment which might not be appropriate for a browser sample, so happy to have a reviewer suggest another approach.

## Type of Change

Please delete options that are not relevant.

- [x] Improve helpefulness of sample in readme 

## Test Coverage
No test

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [N/A ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [N/A ] I have added tests that prove my fix is effective or that my feature works
- [ N/A] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/internal-plugin-mercury/README.md [hunk 1]
```diff
@@ -39,18 +39,22 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
...
-	  },
-	 ...
-	}
-});
+// init webex
+let config = {
+  credentials: process.env.TOKEN
+  //...and anything else you need
+};
+let proxyUrl = process.env.http_proxy || null;
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/internal-plugin-mercury/README.md:50]: Typically, our env var names are in upper case, so:
```
process.env.http_proxy
```
should be:
```
process.env.HTTP_PROXY
```

File: packages/node_modules/@webex/internal-plugin-mercury/README.md [hunk 2]
```diff
@@ -39,18 +39,22 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
...
+if (proxyUrl) {
+  let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+  config.defaultMercuryOptions = {
+    agent: httpsProxyAgent
+  };
+}
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/internal-plugin-mercury/README.md:56]: a new line after this if statement bracket could help improve readability.

File: packages/node_modules/@webex/internal-plugin-mercury/README.md [hunk 3]
```diff
@@ -39,18 +39,22 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
...
-		agent: httpsProxyAgent
-	  },
-	 ...
-	}
-});
+// init webex
+let config = {
+  credentials: process.env.TOKEN
+  //...and anything else you need
+};
```
Comment (InteractiveTimmy) [inline packages/node_modules/@webex/internal-plugin-mercury/README.md:49]: a new line after this if statement bracket could help improve readability.

File: packages/node_modules/@webex/internal-plugin-mercury/README.md (hunk 1 above)
Comment (jpjpjp) [reply packages/node_modules/@webex/internal-plugin-mercury/README.md:50]: OK.

File: packages/node_modules/@webex/internal-plugin-mercury/README.md (hunk 2 above)
Comment (jpjpjp) [reply packages/node_modules/@webex/internal-plugin-mercury/README.md:56]: OK

File: packages/node_modules/@webex/internal-plugin-mercury/README.md (hunk 3 above)
Comment (jpjpjp) [reply packages/node_modules/@webex/internal-plugin-mercury/README.md:49]: OK

**Comment by taymoork2** [comment]
@lalli-flores I'll create a PR to update all references to 2019

File: packages/node_modules/@webex/internal-plugin-mercury/README.md [hunk 4]
```diff
@@ -39,18 +41,25 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
...
+let proxyUrl = process.env.HTTP_PROXY || null;
+
+if (proxyUrl) {
+  let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+  config.defaultMercuryOptions = {
+    agent: httpsProxyAgent
+  };
+}
+
+webex.init(config);
```
Comment (adamweeks) [inline packages/node_modules/@webex/internal-plugin-mercury/README.md:62]: Have you verified that this works? I think the shape of the object to provide an access token to `init` should be:

```js
webex.init({
  credentials: process.env.TOKEN
  config: {
    defaultMercuryOptions: {
      agent: httpsProxyAgent
    }
  }
});
```

**Comment by lalli-flores** [comment]
> @lalli-flores I'll create a PR to update all references to 2019

Works for me

File: packages/node_modules/@webex/internal-plugin-mercury/README.md [hunk 5]
```diff
@@ -39,18 +41,25 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
 
 ```js
 const webex = require(`webex`);
+const url = require('url');
 const HttpsProxyAgent = require('https-proxy-agent');
 
-let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+// init webex
+let config = {
+  credentials: process.env.TOKEN
```
Comment (adamweeks) [inline packages/node_modules/@webex/internal-plugin-mercury/README.md:49]: This line needs to be removed.

File: packages/node_modules/@webex/internal-plugin-mercury/README.md (hunk 4 above)
Comment (adamweeks) [reply packages/node_modules/@webex/internal-plugin-mercury/README.md:62]: Confirmed, you need the credentials token at the same level as "config".

Comment (jpjpjp) [reply packages/node_modules/@webex/internal-plugin-mercury/README.md:62]: OK

**Comment by taymoork2** [comment]
Closed on request by @jpjpjp

**Pull request closed without merging**